### PR TITLE
Show databaseId, even for databases this user owns

### PR DESCRIPTION
**Pull request by shamblesides**

### CODE REVIEW

File: src/userbase-js/src/db.js [hunk 1]
```diff
@@ -1815,6 +1822,7 @@ const shareDatabase = async (params) => {
       case 'DatabaseIdCannotBeBlank':
       case 'DatabaseIdInvalidLength':
       case 'DatabaseIdNotAllowed':
+      case 'DatabaseIdNotAllowedForOwnDatabase':
```
Comment (j-berman) [inline src/userbase-js/src/db.js:1825]: This should work too

File: src/userbase-js/src/db.js [hunk 2]
```diff
@@ -1909,6 +1917,7 @@ const modifyDatabasePermissions = async (params) => {
...
```
Comment (j-berman) [inline src/userbase-js/src/db.js:1920]: This should work too

File: src/userbase-js/src/db.js [hunk 3]
```diff
@@ -723,6 +726,7 @@ const insertItem = async (params) => {
...
```
Comment (j-berman) [inline src/userbase-js/src/db.js:729]: Don't need here or in updateItem, deleteItem, putTransaction, uploadFile, or getFile (will throw DatabaseNotOpen error bc can't open the database)

**Review by j-berman — COMMENTED** [review]
Nice error language, few comments

File: src/userbase-js/src/db.js [hunk 4]
```diff
@@ -668,6 +670,7 @@ const openDatabase = async (params) => {
...
```
[the remark this answers is not in the document]
Comment (shamblesides) [reply src/userbase-js/src/db.js:673]: Gotcha. Resolved all of them

File: src/userbase-js/src/db.js [hunk 5]
```diff
@@ -1617,6 +1623,7 @@ const getDatabases = async (params) => {
...
```
[the remark this answers is not in the document]
Comment (shamblesides) [reply src/userbase-js/src/db.js]: Added!

File: src/userbase-js/src/db.js (hunk 1 above)
Comment (shamblesides) [reply src/userbase-js/src/db.js]: Done

File: src/userbase-js/src/db.js (hunk 2 above)
Comment (shamblesides) [reply src/userbase-js/src/db.js]: Done

File: src/userbase-js/src/db.js (hunk 3 above)
Comment (shamblesides) [reply src/userbase-js/src/db.js]: All done

File: cypress/integration/db.spec.js [hunk 6]
```diff
@@ -1325,6 +1327,34 @@ describe('DB Tests', function () {
         await this.test.userbase.deleteUser()
       })
 
+      it.only('Get 1 Database using database ID, for my own database', async function () {
```
Comment (j-berman) [inline cypress/integration/db.spec.js:1330]: .only

**Pull request merged**